doc/source/whatsnew/v1.1.0.rst

```diff
@@ -215,6 +215,67 @@ MultiIndex
         # Common elements are now guaranteed to be ordered by the left side
         left.intersection(right, sort=False)
 
+- Bug in :meth:`MultiIndex.get_indexer` incorrectly handling use of pad and backfill options.  This is a regression introduced in pandas 0.23.0 (:issue:`29896`)
+
+*pandas [0.23.0, 1.1.0)*
+
+.. code-block:: python
+
+    >>> df = pd.DataFrame({
```

Comment:
use an ipython block to execute & show the df (leave the reindex and so on);

use ipython prompts rather than ```>>>``` see other whatsnew sections

Reply:
a bit confused here -- presumably there's only one pandas environment in which it can execute this code?  It would make sense if that were the current version and so I could use that for the 1.1.0 block but I'm not sure how to programmatically get the docs to execute a code block on an older pandas version

Reply:
for the previous  section, we use static code, e.g. you execute it yourself in an older environment and just paste the output, the build *does* not execute it (hence the code-block), use ipython based formatting (see other entries in the whatsnew on how to do this).